cves/CVE-2008-2364.yml

```diff
@@ -81,11 +83,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: Definitely not. An unlimited amount of interim responses were allowed prior to the fix.
+  code: true
```

Comment:
You say `true` here, but in your answer, you say definitely not. I think this line is asking you to specify whether or not the code was unit tested :)